mapproxy/cache/path.py

```diff
@@ -75,10 +91,17 @@ def tile_location_tc(tile, cache_dir, file_ext, create_dir=False):
     >>> tile_location_tc(Tile((3, 4, 2)), '/tmp/cache', 'png').replace('\\\\', '/')
     '/tmp/cache/02/000/000/003/000/000/004.png'
     """
+    if dimensions is not None:
+        items = list(dimensions.keys())
+        items.sort()
+        dimensions_str = ['{key}-{value}'.format(key=i, value=dimensions[i].replace('/', '_')) for i in items]
+        cache_dir = os.path.join(cache_dir, '_'.join(dimensions_str))
+
```

Comment:
I would recommend to move this part to ` FileCache.tile_location`:
https://github.com/mapproxy/mapproxy/blob/6e3dce33a31d38ddf358c56e69532bb828a3f856/mapproxy/cache/file.py#L48
it will enable support other directory layouts (e.g. `quadkey`)